test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -586,11 +595,10 @@ def test_word_language_model_RNN_TANH(self):
     def test_word_language_model_RNN_RELU(self):
```

Comment:
Can we enable test_word_language_model_RNN_RELU and test_word_language_model_RNN_TANH ?

Reply:
This two tests are blcoked by **General Union types are not currently supported.** error. There seems to be no good solution yet. And it has been added in VSO.